assets/_sass/main.scss

```
@@ -529,16 +530,23 @@ $tech-icon-breakpoint-2: 39rem;
 
 .sponsor__block {
     text-align: center;
+    margin: $space-small;
 }
 
 .sponsor__logo {
+    width: 100%;
+    height: 100%;
     max-width: 18em;
```

I have no idea if this is the right way to do it but it works?
```suggestion
    max-width: 18em;
    min-width: 18em;
```